resources/factions/usa_1970.json

```diff
@@ -57,19 +63,27 @@
   ],
   "naval_units": [
     "FFG Oliver Hazard Perry",
-    "CV-59 Forrestal"
+    "CV-59 Forrestal",
+    "USS Allen M. Sumner (DD-692)",
+    "USS Fletcher FRAM II Destroyer",
+    "USS Laffey (DD-724)",
+    "USS Maddox (DD-731)",
+    "USS The Sullivans (DD-537)"

```

Comment:
Missing CVA-31 here?

Reply:
I put the CV under carrier_names. I do not fully understand the difference between naval_units and carrier_names. Should the CV be under both?

Reply:
Carrier names are simply names that are assigned to the carrier, but you'd still need to mention the unit type under naval unit, i.e. using the name as defined in the carrier's yaml file. But there is indeed something that need to be added in the code. I believe all that's needed is to add the class in the following file around line 1410: https://github.com/dcs-retribution/dcs-retribution/blob/dev/game/theater/controlpoint.py

Now I might still be missing something, so let me know if you run into any trouble 😅